src/Tools/RepoUtil/ConflictingPackagesException.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace RepoUtil
+{
+    internal class ConflictingPackagesException : Exception
```

Comment:
My initial thought was to catch this exception however I suspect that there are other cases where exceptions go unhandled causing that app to crash which isn't a good user experience.  Now that said I could change this such that we catch this exception followed by a catch for System.Exception with a generic "something unexpected happened..." type message.  How does that sound?